requirements-dev.txt

```diff
@@ -5,7 +5,8 @@ click==7.0
 coverage==5.3
 cycler==0.10.0
 decorator==4.4.2
-esse==2021.09.09
+#esse==2021.09.09
```

Comment:
https://exabyte.atlassian.net/wiki/spaces/DGS/pages/2422472713/Including+Intermediate+Changes+in+Dependencies#From-GitHub